src/modules/pos/validator/components/StakeForm/StakeForm.js

```diff
@@ -231,9 +284,9 @@ const StakeForm = ({ t, stakes, account, isStakingTxPending, nextStep, history,
                   />
```

Comment:
Not related to this line:

Maybe we should rename the `buttonTitle` label to 
`buttonTitle={showEmptyState ? t('Continue to validator list') : t('Continue')}`
Since that seems to be where it is navigating. 

I dont know in which scenario we show this component when showEmptyState. Do you know how to test that flow?